index.html

```diff
@@ -13,6 +13,7 @@ <h1>Skillcrush 105 - Advice for Newbies!</h1>
     <section>
         <div class="all-advice-links">
             <div class="skillcrush">
+                <a href="emilyadvice.html" target="_blank">J0nathan's Advice</a><br>
```

Comment:
Hey there! Because your advice file is inside the 'advice' folder - we'll need that `href` attribute to point to `advice/emilyadvice.html` instead of just `emilyadvice.html`